server/src/main/java/org/opensearch/search/sandboxing/ResourceUsageData.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.search.sandboxing;
+
+import java.util.Map;
+
+public class ResourceUsageData {
```

Comment:
Do we really need this class ? I would rather have this as a member variable of all the data related to sandbox.